### DISCUSSION TITLE
Assets outside "src" not loading in build

### DISCUSSION THREAD

**Discussion by Andy-Roger-Kats** [mozilla/hubs#3157] [Hubs Cloud Questions] [answered]
We have a folder where we are keeping our custom assets which sits outside of the main Hubs Cloud "src" folder. Our folder holds all of our custom assets: pngs, jpgs, glbs, etc... When we run locally using "npm start" these assets load fine but when we deploy to the Hubs Cloud server they get 404s. I think this is an issue with the webpack configuration not including assets from outside of the root/src path.

**Comment by pattersonbl2** [comment, marked as answer]
Can you included your assets in the src path and tell us how it goes ?  

Also you can host the assets in an S3 bucket, reference the S3 bucket in your code, and add the S3 bucket url to your CORS settings in the admin panel. 

Marking this as answered for now please unmark if you have more information.

**Reply by nupur711** [reply]
I want to load glb model through custom code, tried adding through <a-entity> in hubs.html but not able to see the output. 3d Model is kept inside assets folder inside src folder. Not getting any error. However, able to see output when creating with <a-box> not with gltf model entity.
